packages/@vuepress/theme-default/components/Page.vue

```diff
@@ -169,6 +163,29 @@ function resolveNext (page, items) {
   return find(page, items, 1)
 }
 
+function resolvePageLink (linkType, { $themeConfig, $page, $route, $site, sidebarItems }) {
+  const resolveLink = linkType === LINK_TYPES.NEXT ? resolveNext : resolvePrev
+
+  // Get link config from theme
+  const { nextLinks, prevLinks } = $themeConfig
+  const themeLinkConfig = linkType === LINK_TYPES.NEXT ? nextLinks : prevLinks
+
+  // Get link config from current page
+  const { next, prev } = $page.frontmatter
+  const pageLinkConfig = linkType === LINK_TYPES.NEXT ? next : prev
+
+  // Page link config will overwrite global theme link config if defined
+  const link = isNil(pageLinkConfig) ? themeLinkConfig : pageLinkConfig
+
+  if (link === false) {
```

Comment:
```suggestion
  if (!link) {
```

Reply:
`link` default value is `undefined`. But we want pages links to be enabled by default, this is why we need to check type **and** value here.

`undefined` value should keep being handled by the following:
```
else {
    return resolveLink($page, sidebarItems)
}
```